lib/rabbit_feed.rb

```diff
@@ -49,6 +50,10 @@ def default_logger
     end
   end
 
+  def route_prefix_extension
```

Comment:
Was there a compelling reason to define this in the `RabbitFeed` module? I ask because I think that moving this parameter to the configuration file might make this parameter easier to set and manage per environment. Also, this might prevent the need for the extra command line parameter as Pooka also reads from `rabbit_feed.yml`. Apologies for the last minute comment, but this just came to mind.